packages/perseus/src/widgets/interactive-graphs/graphs/use-draggable.test.tsx

```diff
@@ -0,0 +1,78 @@
+import {render, screen} from "@testing-library/react";
+import {userEvent as userEventLib} from "@testing-library/user-event";
+import {Mafs} from "mafs";
+import * as React from "react";
+import {useRef} from "react";
+
+import {useDraggable} from "./use-draggable";
+
+import type {vec} from "mafs";
+
+function TestDraggable(props: {
+    point: vec.Vector2;
+    constrain: (point: vec.Vector2) => vec.Vector2;
+}) {
+    const gestureTarget = useRef<HTMLParagraphElement>(null);
+    const {dragging} = useDraggable({
+        ...props,
+        gestureTarget,
+        onMove: () => {},
+    });
+    return (
+        <p role="button" ref={gestureTarget}>
+            dragging: {String(dragging)}
+        </p>
```

Comment:
Question: Is it okay to set the role of a `p` element to "button" like this? I know this is only a test, so maybe it's not super important.